Pipfile

```diff
@@ -26,8 +26,10 @@ sentry-sdk = "~=1.3.1"
 rq = "~=1.9.0"
 wagtail = "~=2.14.1"
 django-lti-provider = "~=0.4.4"
+lac = "==2.1.2"
 
 [dev-packages]
+debugpy = "~=1.4.3"
```

Comment:
I wanted to add watchmedo via the [`watchdog` package](https://github.com/gorakhargosh/watchdog) to manage RQ worker reloading, but apparently [pipenv can't handle `extras` (optional dependencies) properly via the `Pipfile.lock`](https://github.com/pypa/pipenv/issues/4407) just yet.